pyrdp/player/ReplayWindow.py

```diff
@@ -17,16 +19,22 @@ class ReplayWindow(BaseWindow):
     Class for managing replay tabs.
     """
 
-    def __init__(self, options: Dict[str, object], parent: QWidget = None):
-        super().__init__(options, parent)
+    def __init__(self, options: Dict[str, object], mainWindow: MainWindow, parent: QWidget):
+        super().__init__(options, parent=parent)
+        self.mainWindow = mainWindow
 
     def openFile(self, fileName: str, autoplay: bool = False):
         """
         Open a replay file and open a new tab.
         :param fileName: replay path.
         """
-        tab = ReplayTab(fileName)
+        tab = ReplayTab(fileName, parent=self)
         self.addTab(tab, fileName)
         self.log.debug("Loading replay file %(arg1)s", {"arg1": fileName})
         if autoplay:
             tab.play()
+
+    def resizeEvent(self, event: QResizeEvent):
+        super().resizeEvent(event)
+        for i in range(self.count()):
+            self.widget(i).parentResized(event.size().width(), event.size().height())
```

Comment:
You could just pass the event here, since all of these are widgets anyway.

Reply:
I don't even actually use these arguments, I put them because I thought it would be a bad api for the children if i didnt put them. Should I remove them?

Reply:
If the purpose of the code is to drill the event down the widget tree for children to be notified of a parent's resize, it makes sense to have the event as an argument. 

Otherwise don't make the code generic and just make a method that works directly on the `ReplayTab`s that's called `resize` or something.

Reply:
i went with the first approach